salt/returners/redis_return.py

```diff
@@ -150,7 +209,7 @@ def save_load(jid, load, minions=None):
     Save the load to the specified jid
     '''
     serv = _get_serv(ret=None)
-    serv.setex('load:{0}'.format(jid), json.dumps(load), _get_ttl())
+    serv.setex('load:{0}'.format(jid), _get_ttl(), json.dumps(load))
```

Comment:
This goes with the switch to StrictRedis and StrictRedisCluster in the PR. The order of the args to setex is different when using StrictRedis. The cache module was able to use Strict without issues so I used it here as well and only needed this one change.

From the [redis-py docs](https://github.com/andymccurdy/redis-py/blob/master/README.rst):

> redis-py exposes two client classes that implement these commands. The StrictRedis class attempts to adhere to the official command syntax. 

Applicable difference:

> SETEX: Order of 'time' and 'value' arguments reversed.